### ISSUE TITLE
Auto-detect to install 32bit or 64bit

### ISSUE THREAD

**Issue by joedf** [RussellHaley/PUC-Lua-Installer#1]
I have never used wix before, but I have heard of it. That said, one installer should be able to handle both 32bit and 64bit. Also, is it possible have auto-detect the lua version from the binaries?
What does the installer do other than file copying and registry changes?

**Comment by RussellHaley** [comment]
Not quite. A 32 bit installer can't install 64 bit binaries. That's why there are always two different installation packages for Windows. This is normally handled by using two build configurations for the solution. "Auto-detect" would mean reading the lua version from lua.h. 

The main purpose of an MSI installer is to register components in a standardized way to ensure that all parts of the application are removed successfully when the user wants them removed. This includes registry items, shortcuts, resources, pre-requisites and files. The WIX installer I created uses the add/remove utilities in MSI. I believe having an installer will "legitimize" Lua in many Windows users minds. 

My ultimate goal is to create a full Visual Studio solution for Lua, which would bring the source code to a new audience. However, If you are interested in using this project and hosting the output, I will look into adding 64 bit support. The auto-detect will require some file parsing, but is a good feature that I will work on.

**Comment by joedf** [comment]
I see... The MSI installers (32bit and 64bit) could be wrapped together in an exe as a workaround.
Also after your recent changes, the project seems a lot nicer and easier to follow 👍
